pipeline_api/src/main/java/com/splicemachine/pipeline/utils/PipelineUtils.java

```diff
@@ -90,8 +90,8 @@ public static long getPauseTime(final int tries, final long pause) {
             ntries = RETRY_BACKOFF.length - 1;
         }
         long normalPause = pause * RETRY_BACKOFF[ntries];
-        long jitter =  (long)(normalPause * RANDOM.nextFloat() * 0.01f); // 1% possible jitter
-        return normalPause + jitter;
+        // DB-7022 avoid coordinated pauses from multiple threads
+        return RANDOM.nextLong() % normalPause;
```

Comment:
The Random.nextLong() may return a negative value, so need to use the absolute value or something.  See the following for a potential problem with getting the absolute value of the smallest possible long:
https://help.semmle.com/wiki/display/JAVA/Incorrect+absolute+value+of+random+number